### PR TITLE
[FIX] jupyterlab css clobbers downstream css

### PR DESCRIPTION
**Pull request by stevejpurves**
This is a potential fix for issue #464.

Here I have avoided importing the `@jupyterlab/rendermime/style/index.css` file directly and instead import the dependencies listed in that index file, while omitting the files that eventually load the `@jupyterlab/ui-components/style/index.css` file.

I am not sure that we need any styles from `@jupyterlab/ui-components` at all but have made a local file `patch.css` that essentially recreates the `@jupyterlab/ui-components/style/index.css` but without the problematic rules on high-level selector. I'm not sure if we need this or not but wanted to align as closely as possible without including the problematic rules.

The sum of this is that we can load most of the css that was loaded previously. When loading the `ui-components` css and other css that is actually dependent on the rules that I have now omitted, there may be issues that need to be resolved.

So far I have tested with all the examples in the docs and they appear to all render correctly.

I'm interested in what other testing we could do here to ensure that this fixes the issue without causing further downstream issues? cc @minrk @choldgraf 

@moorepants @TimStewartJ  I was thinking that testing for any impact on the LibreTexts build would be a good further validation for starters?

### CODE REVIEW

File: src/patch.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
+/*-----------------------------------------------------------------------------
+| Copied from : https://raw.githubusercontent.com/jupyterlab/jupyterlab/master/packages/ui-components/style/base.css
+| with high level css selectors removed
+|
+| Note: this is highly fragile and may change each release of jupyterlab
```
Comment (choldgraf) [inline src/patch.css:5]:
```suggestion
| Note: this is highly fragile and may change each release of jupyterlab
| See this issue for some context: https://github.com/executablebooks/thebe/issues/464
```

**Review by choldgraf — COMMENTED** [review]
I think that this looks good to me. My assumption is that this won't make a huge difference because (I think) we don't re-use a ton of the lab UI components anyway.

I'm +1 as long as @moorepants confirms that this does not disrupt libretext builds in any way!

**Comment by moorepants** [comment]
@TimStewartJ could you test this in the ckeditor plugin with our examples? We can check on libretexts after it is released.

**Comment by TimStewartJ** [comment]
It appears to be working! I've gone ahead and made a quick patch to ckeditor-binder-plugin that uses this version that can be seen here: https://query.libretexts.org/Sandboxes/jupyterteam_at_ucdavis.edu.

**Comment by moorepants** [comment]
Great, thanks Tim!

File: src/patch.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
+/*-----------------------------------------------------------------------------
```
Comment (moorepants) [inline src/patch.css:1]: I believe we need to include JupyterLab's license in this repo if we copy code from there.

Comment (stevejpurves) [reply src/patch.css:1]: yes, you're right. I really wonder if we need this patch though? I think I'll do some more testing with it removed, and if it stays we'll bring the in license.

**Comment by stevejpurves** [comment]
I've gone ahead and removed the `patch.css` file after testing locally with the examples we have. I see no negative impact from removing it.

@moorepants @TimStewartJ I suggest that we bring this in but @TimStewartJ if you wanted to give it another quick double check, on your code that might be good. Sorry about the churn!

**Comment by TimStewartJ** [comment]
Yep, sure thing. I've deployed this new patch to the same place and tested it. Everything seems to work as intended still.

**Comment by stevejpurves** [comment]
great @TimStewartJ !

**Pull request merged**